doc/src/vpr/debug_aids.rst

```diff
@@ -31,7 +31,8 @@ If you want to read a text file describing the entire routing resource graph, ca
 Placer and Router Debugger
 ==========================
 
-.. image:: https://github.com/verilog-to-routing/verilog-to-routing.github.io/blob/master/img/debuggerWindow.png
+.. image:: https://www.verilogtorouting.org/img/debuggerWindow.png
```

Comment:
You should just put these images in this directory and reference them with `.. image:: debuggerWindow.png` (or `.. figure::` as shown at https://github.com/verilog-to-routing/vtr-verilog-to-routing/blame/e5ff75cc76f83ee2a7a5c4bbda0a278e6980239c/doc/src/vpr/command_line_usage.rst#L1467.

Reply:
No, I remember that the images wouldn't show up when I put their actual directory link. And that this is the general format https://www.verilogtorouting.org/img/*image name*. I had this problem with the Graphics documentation too and just went back and checked to see how I had fixed it (those images also follow this format). The images do show up in the preview now.

Reply:
I'm not sure why one of them isn't showing up. I double checked the name's spelling and it is located in the same directory as the other one.